### ISSUE TITLE
Implement Taxi Generation

### ISSUE THREAD

**Issue by abrac** [lcodeca/SUMOActivityGen#12]
This issue tracks the implementation of taxi generation.

Targets (in order of priority):
- Sep 15:
    - [x] Familiarise myself with the structure of the existing source code
- To be scheduled:
    - [x] Extract taxi stops from OSM
    - [ ] Complete taxi generation to/from any parking area
    - [ ] Complete taxi generation to/from taxi stops

**Comment by abrac** [comment]
I will work on this issue once @lcodeca pushes her private branch  (#11)

**Comment by lcodeca** [comment]
Thank you @abracwan, I just pushed the branch https://github.com/lcodeca/SUMOActivityGen/tree/taxi-fleet.

I'm going to update this issue with my taxi-fleet plan as soon as I can.

**Comment by abrac** [comment]
@lcodeca 

I've been trying to find time to complete task 1 (i.e. "Familiarise myself with the structure of the existing source code"), but unfortunately I haven't looked at it because of other deadlines 😞. 

My supervisor recommended that I don't focus on the generation of taxis through SAGA as we have found a simpler way of simulating our taxis. I'd still like to help on this, but will do so on a part-time basis. I will extend the deadline of task 1 to Sep 15. 

I am sure I will be able to complete it by then. I hope this hasn't inconvenienced you...

**Comment by abrac** [comment]
@lcodeca Sorry, I'm going to close this issue for now. I have indeed spent some time looking through the code, but I don't have much motivation to work on the rest of the issue due to the fact that I don't need the functionality anymore. 

I initially wanted to generate taxis automatically using SAGA, but I was able to generate their routes by parsing measured GPS traces of a number taxis in the city I am studying. I *might* look into this again, but it's much lower in my to-do list now.
